lib/archethic/bootstrap/sync.ex

```diff
@@ -133,20 +135,27 @@ defmodule Archethic.Bootstrap.Sync do
   @doc """
   Fetch and load the nodes list
   """
-  @spec load_node_list(list(Node.t())) :: :ok | {:error, :network_issue}
-  def load_node_list([node | rest]) do
-    case P2P.send_message(node, %ListNodes{}) do
+  @spec load_node_list() :: :ok | {:error, :network_issue}
+  def load_node_list() do
```

Comment:
I actually used the `nodes` arg you removed in: https://github.com/archethic-foundation/archethic-node/pull/933/files#diff-0eae82f6d9ce2a7425cff87a7358eba0b9f634562799f77933fc063411658af8R239 . 

What do you think, is it not relevant to be able to request from only some nodes?

Reply:
In is using a quorum to request the node list, so the filter you did on the node in your PR is almost the same done by the quorum.
And quorum provide more trustable result than a single node query